zanata-common-api/src/main/java/org/zanata/rest/service/AsynchronousProcessResource.java

```diff
@@ -46,9 +47,11 @@
  * @author Carlos Munoz <a
  *         href="mailto:camunoz@redhat.com">camunoz@redhat.com</a>
  */
+@Path(AsynchronousProcessResource.SERVICE_PATH)
+@ExternallyManagedLifecycle
```

Comment:
It's there to make Enunciate stop complaining - it can get worried when you
put annotations on interfaces instead of classes, because it thinks like a
runtime, not a doclet tool.

On 29 March 2016 at 15:40, Carlos Munoz <notifications@github.com> wrote:

> In
> zanata-common-api/src/main/java/org/zanata/rest/service/AsynchronousProcessResource.java
> <https://github.com/zanata/zanata-api/pull/45#discussion_r57674094>:
>
> > @@ -46,9 +47,11 @@
> >   * @author Carlos Munoz <a
> >   *         href="mailto:camunoz@redhat.com">camunoz@redhat.com</a>
> >   */
> > +@Path(AsynchronousProcessResource.SERVICE_PATH)
> > +@ExternallyManagedLifecycle
>
> What does this annotation do exactly?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-api/pull/45/files/88c69b8ab5ebf768bc2c3efd868b90d2a3cfe951#r57674094>
>



-- 
Sean Flanigan

Principal Software Engineer
Globalisation Tools Engineering
Red Hat